lib/filters/core-filters.js

```diff
@@ -2,14 +2,39 @@
 const { runWhenEnvIsAvailable, external } = require('./api')
 const { addFilter, getFilter } = external
 
-/**
- * Logs an object in the template to the console in the browser.
- * @param  {Any} a any type
- * @return {String} a script tag with a console.log call.
- * @example {{ "hello world" | log }}
- * @example {{ "hello world" | log | safe }}  [for environments with autoescaping turned on]
- */
 runWhenEnvIsAvailable(() => {
   const nunjucksSafe = getFilter('safe')
+
+  /**
+   * Logs an object in the template to the console in the browser.
+   * @param  {Any} a any type
```

Comment:
Tried this one too and it works with `any` or `unknown` (both lowercase) but not `Any` (titlecase)

```suggestion
   * @param  {any} a - any type
```

<br>

We'll want to remove the example with `| safe` since `nunjucksSafe()` is always used now

```mjs
  addFilter(
    'log',
  
    /**
     * Logs an object in the template to the console in the browser.
     *
     * @param {any} a - any type
     * @returns {string} a script tag with a console.log call.
     * @example
     * ```njk
     * {{ "hello world" | log }}
     * ```
     */
    (a) =>
      nunjucksSafe(
        `<script>console.log(${JSON.stringify(a, null, '\t')});</script>`
      )
  )
```